kernel/sucompat.c

```diff
@@ -246,11 +246,7 @@ static struct kprobe newfstatat_kp = {
 };
 
 static struct kprobe execve_kp = {
-#if LINUX_VERSION_CODE >= KERNEL_VERSION(5, 9, 0)
```

Comment:
Add version check

Reply:
I think the version check need change.Because in some version.(Such as 4.14)It use do_execveat_common

Reply:
how about?
4.19 - 5.9: __do_execve_file
5.9+: do_execveat_common
4.14 - 4.19: do_execveat_common

Reply:
I think it's ok